prov/verbs/src/ep_rdm/verbs_rdm_cm.c

```diff
@@ -78,9 +78,8 @@ fi_ibv_rdm_batch_repost_receives(struct fi_ibv_rdm_conn *conn,
 {
 	const size_t idx = (conn->cm_role == FI_VERBS_CM_SELF) ? 1 : 0;
 	struct ibv_recv_wr *bad_wr = NULL;
-	struct ibv_recv_wr wr[num_to_post];
+	struct ibv_recv_wr wr[num_to_post + 1];
```

Comment:
Why would we want the array size larger than the number to post?

Reply:
My goal was to not add overhead and not [go beyond the WR's array](https://github.com/ofiwg/libfabric/blob/d61f9cb9043c85cc8c072d414f0ec8f0017d9328/prov/verbs/src/ep_rdm/verbs_rdm_cm.c#L103).
But I agree, it  doesn't look natural. I'll re-implement to make it more understandable